src/ripple/app/tx/impl/SetSignerList.cpp

```diff
@@ -305,13 +321,14 @@ SetSignerList::removeSignersFromLedger (Keylet const& accountKeylet,
 }
 
 void
-SetSignerList::writeSignersToSLE (SLE::pointer const& ledgerEntry) const
+SetSignerList::writeSignersToSLE (
+    SLE::pointer const& ledgerEntry, std::uint32_t flags) const
 {
-    // Assign the quorum.
+    // Assign the quorum, default SignerListID, and flags.
     ledgerEntry->setFieldU32 (sfSignerQuorum, quorum_);
-
-    // For now, assign the default SignerListID.
     ledgerEntry->setFieldU32 (sfSignerListID, defaultSignerListID_);
+    if (flags) // If the flags are zero default them (they default to zeroed).
```

Comment:
The comment is a little weirdly worded and I’m not sure it really adds a lot. I think that “non-default flags specified” works better.

Reply:
Good point.  I've revised the wording on this comment.  Thanks.